notarize.js

```diff
@@ -334,21 +334,36 @@ const createNewRevision = async (
   let verificationData = {
     previous_verification_hash: previousVerificationHash,
     nonce: prepareNonce(),
-    //domain_id: getDomainName(), // TODO
     local_timestamp: timestamp,
     revision_type,
   }
 
   switch (revision_type) {
     case "content":
-      const fileContent = fs.readFileSync(filename, "utf8")
```

Comment:
You just deleted the "content" case.

Reply:
That's not good. Will fix.

Reply:
Oh, this is standard JS syntax that I'm not aware of. I'm used to a more explicit syntax in another language.